src/main/java/it/smartcommunitylab/aac/webauthn/provider/WebAuthnAccountService.java

```diff
@@ -0,0 +1,242 @@
+package it.smartcommunitylab.aac.webauthn.provider;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.yubico.webauthn.data.ByteArray;
```

Comment:
remove the usage of this outside webauthn specific services

Reply:
Done with 42b67e746e5f71e7ec4b3a5676cec60d4bd2bf6f